akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -736,8 +736,14 @@ private final case class SavedIslandData(islandGlobalOffset: Int, lastVisitedOff
         fuseIntoExistingInterperter(shell)
 
       case _ ⇒
+
+        val dispatcher =
+          effectiveAttributes.mandatoryAttribute[ActorAttributes.Dispatcher] match {
+            case ActorAttributes.IODispatcher           ⇒ settings.blockingIoDispatcher
+            case ActorAttributes.Dispatcher(dispatcher) ⇒ dispatcher
+          }
         val props = ActorGraphInterpreter.props(shell)
-          .withDispatcher(effectiveAttributes.mandatoryAttribute[ActorAttributes.Dispatcher].dispatcher)
+          .withDispatcher(dispatcher)
```

Comment:
There is also a `.withDispatcher` in ActorMaterializerImpl.scala

Reply:
and another one in this file for TLSActor
and IOSinks and a lookup in OutputStreamSourceStage

Reply:
Thanks, I didn't think of those special stages, I'll double check there aren't others.

Reply:
Checked but forgot to add back that I did, checked and ok